tests/engine/path_helper.py

```diff
@@ -142,6 +146,95 @@ def testGetRelativePathForPathSpec(self):
         qcow_path_spec)
     self.assertIsNone(display_name)
 
+  def testAppendPathEntries(self):
+    """Tests the AppendPathEntries function."""
+    separator = '\\'
+    path = '\\Windows\\Test'
+
+    # Test depth of ten skipping first entry.
+    # The path will have 9 entries as the default depth for ** is 10, but the
+    # first entry is being skipped.
+    count = 10
+    skip_first = True
+    paths = path_helper.PathHelper.AppendPathEntries(
+        path, separator, count, skip_first)
+
+    # Nine paths returned
+    self.assertEqual(len(paths), 9)
+
+    # Nine paths in total, each one level deeper than the previous.
+    check_paths = [
+        '\\Windows\\Test\\*\\*',
+        '\\Windows\\Test\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*\\*\\*\\*\\*',
+        '\\Windows\\Test\\*\\*\\*\\*\\*\\*\\*\\*\\*\\*']
+    self.assertItemsEqual(paths, check_paths)
```

Comment:
not portable to python 3
`Instance of 'PathHelperTest' has no 'assertItemsEqual' member (no-member)`